osgp/protocol-adapter-iec60870/osgp-iec60870/src/main/java/org/opensmartgridplatform/iec60870/Iec60870ConnectionRegistry.java

```diff
@@ -0,0 +1,30 @@
+package org.opensmartgridplatform.iec60870;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.openmuc.j60870.Connection;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Iec60870ConnectionRegistry {
+    private Map<Integer, Connection> connections = new ConcurrentHashMap<>();
+
+    public List<Connection> getAllConnections() {
+        return new ArrayList<>(this.connections.values());
+    }
+
+    public Connection getConnection(final int id) {
```

Comment:
Connection id was initially used in the Iec60870ServerEventListener class only. As I needed something to keep track of the connections, I've added the connection registry and used this id in the connection registry as well.
Refactored code so that connection id is no longer used and removed this method.